# Sensor ID 0 â€“ 7

uint8 sensor_id
---